### PR TITLE
Style improvements for the abilities page

### LINKED ISSUE #2863: Update Abilities page
- [ ] Make sure that the Abilities page looks the same there as in the designs. 

Screenshot 2023-01-16 at 08 29 19

Screenshot 2023-01-13 at 14 35 51

- [ ] Add the notification bullet on the portfolio icon. A green dot should be added when the user has an ability.

Screenshot 2023-01-16 at 08 31 25

### PR DESCRIPTION
**Pull request by kkosiorowska**
Closes #2863

This PR improves styles for the abilities page. Changes:

- Notification dot on the portfolio icon. A green dot should be added when the user has an ability.
- Update tooltip styles
- Add time details for the ability if it is possible
- Display ENS name or shortened address for the ability
- Small refactor
- Add component tests

## UI
Screenshot 2023-01-20 at 11 06 34

Screenshot 2023-01-20 at 11 07 00

Screenshot 2023-01-20 at 11 07 39

## To Test
- [ ] check styles for tooltips
- [ ] check if the name is displayed correctly on the ability card
- [ ] check that the notification icon works properly

## Testing Env
```
SUPPORT_ABILITIES=true
```

Latest build: [extension-builds-2892](https://github.com/tallyhowallet/extension/suites/10487132256/artifacts/519904459) (as of Fri, 20 Jan 2023 12:40:24 GMT).

### CODE REVIEW

**Review by jagodarybacka — APPROVED** [review]
✅

File: ui/pages/Abilities/__tests__/AbilityCardHeader.test.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import React from "react"
+import {
+  createAccountData,
+  createAccountState,
+  TEST_ADDRESS,
+} from "../../../tests/factories"
+import { renderWithProviders } from "../../../tests/test-utils"
+import AbilityCardHeader from "../AbilityCardHeader"
+
+const ABILITY: Ability = {
```
Comment (jagodarybacka) [inline ui/pages/Abilities/__tests__/AbilityCardHeader.test.tsx:12]: It's duplicated in another test so maybe we can move it to a single file with mocks?

File: ui/pages/Abilities/__tests__/AbilityCard.test.tsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+      <AbilityCard ability={{ ...ABILITY, closeAt: date.toDateString() }} />,
+      {
+        preloadedState: { account: createAccountState() },
+      }
+    )
+    expect(ui.queryByText(TIME_DETAILS.close)).toBeInTheDocument()
+  })
+
+  it("should not display the time detail message when the ability date is not in the next month", async () => {
+    // Start in 5 weeks
```
Comment (jagodarybacka) [inline ui/pages/Abilities/__tests__/AbilityCard.test.tsx:75]: > when the ability date is not in the next month

If it starts in 5 weeks then it is in the next month right?

**Review by jagodarybacka — COMMENTED** [review]
Sorry, I thought I reviewed whole PR but it was just one commit 🙈 Nothing blocking anyway

**Review by jagodarybacka — CHANGES_REQUESTED** [review]
Ah no we have a blocker - you can't use the app with `SUPPORT_ABILITIES=false` because it is throwing errors on install

File: ui/pages/Abilities/__tests__/AbilityCard.test.tsx (hunk 2 above)
Comment (kkosiorowska) [reply ui/pages/Abilities/__tests__/AbilityCard.test.tsx:60]: Let's make the description clearer.

**Review by jagodarybacka — APPROVED** [review]
All good now 👍

**Pull request merged**